api/organisations/invites/views.py

```diff
@@ -59,6 +59,12 @@ def join_organisation_from_link(request, hash):
 
     invite = get_object_or_404(InviteLink, hash=hash)
 
+    if (
```

Comment:
Now that we've updated the logic in the `join_organisation_from_invite` function we should remove this validation (and from the `InviteLinkViewSet.get_queryset` below) or update it to also verify if payments are enabled, before we throw the exception.